Dockerfile

```diff
@@ -11,7 +11,7 @@ WORKDIR /analyzer-lsp
 # TODO limit to prevent unnecessary rebuilds
 COPY  . /analyzer-lsp
 COPY --from=builder /analyzer-lsp/konveyor-analyzer /usr/bin/konveyor-analyzer
+COPY --from=builder /analyzer-lsp/external-providers/golang-external-provider/golang-external-provider /usr/bin/golang-external-provider
 COPY provider_container_settings.json /analyzer-lsp/provider_settings.json
 
-
-CMD ["/bin/bash", "-c", "go run main.go;cat output.yaml"]
+CMD ["/bin/bash", "-c", "golang-external-provider & disown -r; go run main.go; cat output.yaml"]
```

Comment:
interesting, I assumed the plugins would be given to the engine and it would handle starting them up as part of the init process. I think this is fine but maybe a little weird from a UX perspective? I don't love requiring the users to start up all the providers by hand

Reply:
Discussion and update incoming captured in #142